packages/cli/src/commands/org/config/orgConfigBase.ts

```diff
@@ -47,7 +40,7 @@ export default abstract class SnapshotBase extends Command {
   public static hidden = true;
 
   protected async dryRun() {
-    const project = new Project(normalize(this.flags.projectPath));
+    const project = new Project(normalize(this.projectPath));
```

Comment:
So, if the command is ran inside a random folder that is not a valid project, what happens ?

What error will the end user see ?

The message is important to explain "why it's not working", and that it's not a cryptic `throw errror.

Reply:
Right now, there is a generic error message
![image](https://user-images.githubusercontent.com/12199712/122798500-aaf05c00-d28e-11eb-97f6-f247ed3e449b.png)

CDX-354 will provide more information about the error